<?xml version="1.0"?>
<launch>	
	<node pkg="load_transport" name="control" type="main_control_single.py" output="screen">
		<param name="tello_ns" value="$(arg tello_ns)" />
		<param name="lift_thrust" value="$(arg lift_thrust)" />
		<param name="lift_duration" value="$(arg lift_duration)" />
		<param name="pitch_kp" value="$(arg pitch_kp)" />
		<param name="pitch_ki" value="$(arg pitch_ki)" />
		<param name="pitch_kd" value="$(arg pitch_kd)" />
		<param name="yaw_kp" value="$(arg yaw_kp)" />
		<param name="yaw_ki" value="$(arg yaw_ki)" />
		<param name="yaw_kd" value="$(arg yaw_kd)" />
	</node>

	<node pkg="rosbag" type="record" name="recorder" output="screen" if="$(arg is_record)" args="
	record -o $(arg dir_ns)/$(arg bag_name).bag 
	/state_transition 
	/$(arg tello_ns)/cmd_vel 
	/$(arg tello_ns)/land 
	/$(arg tello_ns)/manual_takeoff 
	/$(arg tello_ns)/camera/camera_info 
	/$(arg tello_ns)/camera/compressed/compressed 
	/$(arg tello_ns)/imu 
	/$(arg tello_ns)/odom 
	/$(arg tello_ns)/status 
	/$(arg tello_ns)/bearing/local 
	/$(arg tello_ns)/bearing/global 
	/$(arg tello_ns)/pitch/error 
	/$(arg tello_ns)/yaw/error
	"/>
</launch>
